### ISSUE TITLE
Revisit TinyMCE setup

### ISSUE THREAD

**Issue by gigamorph** [ProjectMirador/mirador#1279]
Right now the assets and plugins for TinyMCE are mixed with those of Mirador proper and only trained eyes can readily distinguish which is for which.

Search path for TinyMCE is currently not set explicitly and is deduced internally using many methods such as looking at `document.location.href`, analyzing the `<script>` tag used to include the TinyMCE main script, and as the last resort, examining `document.currentScript.src`. 

That does not work in certain cases, i.e., when `location` is not useful and `document.currentScript` is not available, such as from an embedded/concatenated script from Drupal in an IE-11 browser. 

So I think we need to
(1) set the path explicitly -- Only the following seemed to work with the current version but I'm not sure
```
tinymce.base = settings.buildPath;  // or whatever path at which TinyMCE assets is found 
tinymce.setup();
```
(2) if possible separate TinyMCE assets in its own folder

**Comment by morpheus-87** [comment]
Is this issue solved in the latest release `2.6.0`, @aeschylus?

**Comment by camillevilla** [comment]
This ticket is being closed as part of the Mirador 3 issues review. We encourage all Mirador 2 implementers to try out Mirador 3 and leave feedback in new tickets.
